import Exception = require('./Exception');

/**
 * 非法调用异常
 */
class InvalidCallException extends Exception {}

export = InvalidCallException;
